### ISSUE TITLE
`Raw` as a synonym for `TSVRaw`

### ISSUE THREAD

**Issue by alexey-milovidov** [ClickHouse/ClickHouse#62079]
**Use case**

```
SELECT format('{0} [ {1} ] ({2}) <{3}> {4}: {5}', event_time_microseconds, thread_id, query_id, level, logger_name, message) AS str
FROM clusterAllReplicas(default, system.text_log)
WHERE event_date >= today() - 2 AND query_id = 'bf73e3a1-d9c4-4e39-b821-27adacc487aa'
ORDER BY event_time_microseconds
FORMAT TSVRaw
```

**Comment by Unalian** [comment]
Hi @alexey-milovidov I would like to work on this issue

**Comment by Blargian** [comment]
Is this still in progress? If not I will do it.

**Comment by Unalian** [comment]
> Is this still in progress? If not I will do it.

Yes. This one is a bit complicated, but I'll finish it this week or next. Let me know if you have any ideas.

**Comment by Blargian** [comment]
@Unalian it looks to me like `TabSeparated` has a synonym `TSVRaw`. See the `registerInputFormatTabSeparated` function for how those names get registered to the format. I didn't have time to look too deeply into it so i'm not 100% sure but it could be as simple as adding another `registerWithNamesAndTypes` call. https://github.com/ClickHouse/ClickHouse/blob/f6c19d5f748cd84f9c2994de63ecc27a282a1c87/src/Processors/Formats/Impl/TabSeparatedRowInputFormat.cpp#L387-L406

**Comment by Unalian** [comment]
> @Unalian it looks to me like `TabSeparated` has a synonym `TSVRaw`. See the `registerInputFormatTabSeparated` function for how those names get registered to the format. I didn't have time to look too deeply into it so i'm not 100% sure but it could be as simple as adding another `registerWithNamesAndTypes` call.
> 
> https://github.com/ClickHouse/ClickHouse/blob/f6c19d5f748cd84f9c2994de63ecc27a282a1c87/src/Processors/Formats/Impl/TabSeparatedRowInputFormat.cpp#L387-L406
> 
> Formats are all registered in `src/Formats/registerFormats.cpp` - debugging where the code goes from those calls can be helpful.

`TabSeparated` has a synonym `TSVRaw` is an amazing hint for me. I originally planned to have FormatFactory inherit IFactoryWithAliases(or achieve some important functions from IFactoryWithAliases), just like `DataTypeFactory` or `FunctionFactory` do. 
>
>
https://github.com/ClickHouse/ClickHouse/blob/f6c19d5f748cd84f9c2994de63ecc27a282a1c87/src/DataTypes/DataTypeFactory.h#L20-L30
>
>
I will evaluate both options, thanks for your reply.

**Comment by Unalian** [comment]
[Pull request](https://github.com/ClickHouse/ClickHouse/pull/63394) has been created.

**Comment by yariks5s** [comment]
Closed by #63394.
